pkg/helpers/newapp/app/dockerimagelookup.go

```diff
@@ -230,16 +230,16 @@ func (s ImageImportSearcher) Search(precise bool, terms ...string) (ComponentMat
 		if image.Status.Status != metav1.StatusSuccess {
 			klog.V(4).Infof("image import failed: %#v", image)
 			switch image.Status.Reason {
-			case metav1.StatusReasonInternalError:
+			case metav1.StatusReasonInternalError, metav1.StatusReasonUnauthorized:
 				// try to find the cause of the internal error
 				if image.Status.Details != nil && len(image.Status.Details.Causes) > 0 {
 					for _, c := range image.Status.Details.Causes {
-						klog.Warningf("container image registry lookup failed: %s", c.Message)
+						klog.Warningf("container image remote registry lookup failed: %s", c.Message)
 					}
 				} else {
-					klog.Warningf("container image registry lookup failed: %s", image.Status.Message)
+					klog.Warningf("container image remote registry lookup failed: %s", image.Status.Message)
 				}
-			case metav1.StatusReasonInvalid, metav1.StatusReasonUnauthorized, metav1.StatusReasonNotFound:
+			case metav1.StatusReasonInvalid, metav1.StatusReasonNotFound:
 			default:
```

Comment:
These image.Status failures do not produce an error - I'm only adding the 'StatusReasonUnauthorized' to the list of reasons to print a warning, but leave the others as/is.  The unauthorized reason is pretty mild, so even if the new-app cmd succeeds to find the same image in local storage, it's not confusing to also see that the remote registry was tried first. User will see the msg: `you may not have access to the container image "registry/image"` so if the cmd ultimately fails, they'll have more idea why, but if it succeeds due to local storage, then that's ok, too, imo.